pom.xml

```diff
@@ -271,6 +271,7 @@
       <version.h2.driver-within-wildfly>1.4.193</version.h2.driver-within-wildfly>
       <version.h2.driver>1.4.180</version.h2.driver>
       <version.hibernate.osgi>${version.hibernate.core}</version.hibernate.osgi>
+      <version.hibernate.v53>5.3.3.Final</version.hibernate.v53>
```

Comment:
We already have `version.hibernate.core` above pointing to 5.3.1.Final. Do you really need 5.3.3?

Reply:
I think that we need two versions of Hibernate. A 5.1.x version to test the feature `infinispan-hibernate-cache-v51` and a 5.3.x version to test the feature `infinispan-hibernate-cache-v53`.

Reply:
My point is not about 5.1.x vs 5.3.x. It's about different 5.3.x versions...

Above in the pom it defines:

    <version.hibernate.core>5.3.1.Final</version.hibernate.core>

And you define:

    <version.hibernate.v53>5.3.3.Final</version.hibernate.v53>

To reiterate, why can't you use 5.3.1.Final?

Reply:
This line should be removed...

Reply:
Actually...

Leave this line as is, so leave `version.hibernate.v53` here, but then make cache-v53 use:

    <version.hibernate.core>${version.hibernate.v53}</version.hibernate.core>

Reply:
By doing that, we fix the Hibernate 5.3.x version in a single place, `version.hibernate.v53`. Although the question begs while you need to add `version.hibernate.v53` to root pom.xml rather than define it in `cache-v53/pom.xml`, keeping all more encapsulated.